app/src/main/java/io/github/droidkaigi/confsched2018/presentation/search/item/SearchResultSpeakerItem.kt

```diff
@@ -19,8 +20,11 @@ data class SearchResultSpeakerItem(
         return ViewHolder(viewDataBinding)
     }
 
+    @SuppressWarnings("deprecation")
     override fun bind(viewBinding: ItemSearchSpeakerBinding, position: Int) {
         viewBinding.searchContent!!.speaker = speaker
+        viewBinding.searchContent!!.speakerName.text = Html.fromHtml(speaker.name)
```

Comment:
I think it's better to bind it by using DataBinding.
How about creating CustomBindingAdapter like `htmlText`?

Reply:
io.github.droidkaigi.confsched2018.presentation.common.binding.TextBinding.kt

```kotlin
@BindingAdapter(value = ["android:htmlText"])
fun TextView.setHtmlText(htmlText: String?) {
    htmlText ?: return

    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
        text = Html.fromHtml(htmlText, Html.FROM_HTML_MODE_LEGACY)
    } else {
        text = Html.fromHtml(htmlText)
    }
    
    text = Html.fromHtml(htmlText)
}
```

Use the custom attribute like this.

```xml
 app:htmlText="@{session.title}"
```

Reply:
Could you write like this? I want to use DataBinding.

```
<data>
<variable name="searchText">
</data>
<TextView
  app:searchText="@{seearchText}"
  android:text="@{speaker.name}"
```

Reply:
Thank you. I will investigate it.

Reply:
In fact, the way @takahirom mentioned is almost same as this PR.
https://github.com/DroidKaigi/conference-app-2018/pull/135/files

But your PR is the first. so we want to merge your PR. Can you fix? 🙇 
Or if you want to prioritize #135, it's okay too! Feel free to tell us!

Reply:
https://github.com/DroidKaigi/conference-app-2018/pull/135/files
looks like better than mine.
Please merge #135.

Reply:
Got it! Thanks for tackling this issue!
We would be really happy if you'll catch another issue 🙏 
Feel free to tackle again! Thanks!